OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -320,8 +320,12 @@ protected function configureOptions(OptionsResolverInterface $resolver)
             'auth_with_one_url'   => false,
         ));
 
-        $resolver->setAllowedValues(array(
-            'csrf' => array(true, false),
-        ));
+        if (version_compare(Kernel::VERSION, '2.6', '>=')) {
```

Comment:
How can I feature detect the possibility to pass a single string instead of an array?